client/src/components/helper/FindTaxRate.js

```diff
@@ -0,0 +1,5 @@
+export function GetTaxRate(bill) {
+  let getTaxRate = (bill.tax_total / bill.subtotal);
```

Comment:
❗ should prevent divide by 0 errors

`if (bill.subtotal === 0) { return 0; }` or something

Reply:
Sounds good. Added it.